tools/exekall/exekall/main.py

```diff
@@ -780,7 +780,7 @@ def handle_cycle(path):
     if rst_expr_list:
         id_kwargs['style'] = 'rst'
         for expr in expr_list:
-            out(expr.get_id(**id_kwargs))
+           out('* {}'.format(expr.get_id(**id_kwargs)))
```

Comment:
not really, that bullet marker used to be added by code that is removed by that commit and replaced by an `exec` reST directive. Otherwise, this commit will generate a paragraph without new lines in the output instead of a bullet list.